back-end/models/categories.model.js

```diff
@@ -5,7 +5,8 @@ var categoriesSchema = new mongoose.Schema({
     type: String,
     unique: true,
     required: true
-  }
+  },
+  description: string
```

Comment:
Sorry for the late reply. Everything looks great except, "string" should be "String". Since "string" is not a valid type the app doesn't start. If you update that, I happily merge it.

Reply:
Thanks for noticing I'll fix that right now